examples/02-Advanced-workflow.ipynb

```diff
@@ -0,0 +1,1060 @@
+{
```

Comment:
<p>Maybe let's call it Advanced NVTabular workflow?</p><p> </p><p>I dont think we do <code>Operators as well as defining an operator of our own.</code> . We should remove it. I think that is an own notebook. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/NVTabular/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
made the change

Reply:
made the change